source/UF/H-Levels.lagda

```diff
@@ -153,11 +154,61 @@ hlevel-closed-under-Π {𝓤} (succ n) X Y m f g =
 
 \end{code}
 
-The subuniverse of types of hlevel n:
+The subuniverse of types of hlevel n is defined as follows.
 
 \begin{code}
 
 ℍ : ℕ → (𝓤 : Universe) → 𝓤 ⁺ ̇
 ℍ n 𝓤 = Σ X ꞉ 𝓤 ̇ , X is-of-hlevel n
 
 \end{code}
+
+From Univalence we can show that (ℍ n) is of level (n + 1), for all n : ℕ.
+
+The next lemma should probably be added as it is the equivalent definition
+of is-prop.
+
+\begin{code}
+
+lemma : {X : 𝓤 ̇} → ((x x' : X) → is-contr (x ＝ x')) → is-prop X
```

Comment:
`lemma` is not a good name for such a general fact. How about `is-prop-criterion`. But if you have a better idea, go ahead. Please just avoid `lemma`.